File: 09-Nov-Noon (SQS, SNS, EventBridge)/sns-python-code/create_subscription.py
import boto3

# Create an SNS client
sns = boto3.client('sns')

# Create email subscription
response = sns.subscribe(TopicArn="arn:aws:sns:us-east-1:1234567890:MyTopic", Protocol="email", Endpoint="ugetaws@gmail.com")
subscription_arn = response["SubscriptionArn"]

print(subscription_arn)

File: 09-Nov-Noon (SQS, SNS, EventBridge)/sns-js-code/create_subscription.js
// Load the AWS SDK for Node.js
var AWS = require('aws-sdk');
// Set region
AWS.config.update({region: 'us-east-1'});

// Create subscribe/email parameters
var params = {
  Protocol: 'EMAIL', /* required */
  TopicArn: 'arn:aws:sns:us-east-1:1234567890:MyTopic', /* required */
  Endpoint: 'ugetaws@gmail.com'
};

// Create promise and SNS service object
var subscribePromise = new AWS.SNS({apiVersion: '2010-03-31'}).subscribe(params).promise();

// Handle promise's fulfilled/rejected states
subscribePromise.then(
  function(data) {
    console.log("Subscription ARN is " + data.SubscriptionArn);
  }).catch(
    function(err) {
    console.error(err, err.stack);
  });
